presto-hive/src/test/java/com/facebook/presto/hive/s3/TestPrestoS3FileSystem.java

```diff
@@ -145,15 +146,16 @@ public void testEndpointWithPinToCurrentRegionConfiguration()
     }
 
     @Test
-    public void testInstanceCredentialsEnabled()
+    public void testAssumeRoleCredentials()
```

Comment:
shall we keep both testInstanceCredentials and testAssumeRoleCredentials?

Reply:
yes, we can.. addressed it.